### PR TITLE
Dialogue

### PR DESCRIPTION
**Pull request by dighjd12**
Worked on Dialogue :)

### CODE REVIEW

**Comment by AEgan** [comment]
to address #20

File: Patient/Assets/GUITest.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	void Start(){
+		updateText("sdfddsds", 3F);
+	}
+
+	void OnGUI() {
+		//GUIStyle style = new GUIStyle ();
+		//style.richText = true;
+		float currentTime = Time.time;
+		if (currentTime-startingTime <= duration) {
+						show = true;
```
Comment (corychang) [inline Patient/Assets/GUITest.cs]: Could you reduce the amount of white space to a single tab?

File: Patient/Assets/GUITest.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				} else {
```
Comment (corychang) [inline Patient/Assets/GUITest.cs]: Same here, could you line up the braces

File: Patient/Assets/GUITest.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	     			show = false;
+		        }
+
+		if (show) {
+			GUI.Label (new Rect (10, 140, 480, 200), dialogue);
+			isDialogOn =true;
+			}
```
Comment (corychang) [inline Patient/Assets/GUITest.cs]: same here

File: Patient/Assets/GUITest.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+		//style.richText = true;
+		float currentTime = Time.time;
+		if (currentTime-startingTime <= duration) {
+						show = true;
+				} else {
+	     			show = false;
+		        }
+
+		if (show) {
+			GUI.Label (new Rect (10, 140, 480, 200), dialogue);
```
Comment (corychang) [inline Patient/Assets/GUITest.cs]: Could you add a comment explaining the magic numbers here? Best practices here would be to make consts at the top of the file and use those here.

File: Patient/Assets/GUITest.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+using System.Collections;
+
+public class GUITest : MonoBehaviour {
+	
+	private bool show = false;
+	private string dialogue;
+	private float startingTime;
+	private float duration;
+	public float Duration{get{return duration;}}
+	private bool isDialogOn = false;
```
Comment (corychang) [inline Patient/Assets/GUITest.cs]: isDialogueOn

**Comment by corychang** [comment]
Looks pretty good, just a few minor things to change :smiley:

**Comment by dighjd12** [comment]
Pushed again to address comments :)

**Pull request merged**